packages/langium/test/lsp/find-references.test.ts

```diff
@@ -0,0 +1,505 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { createLangiumGrammarServices } from '../../src';
+import { expectFindReferences } from '../../src/test';
+
+const grammarServices = createLangiumGrammarServices().grammar;
+const findReferences = expectFindReferences(grammarServices);
+
+describe('findReferences', () => {
+    test('Must find references to parent interface in another interface declaration. Including parent interface declaration', async () => {
+        const grammar = `grammar test hidden (WS)
+        terminal ID: /\\w+/;
+
+        interface <|<|>A|> {
+            name: string
+        }
+        
+        interface B extends <|A|> {}
+        `;
+
+        await findReferences({
+            text: grammar,
+            index: 0,
+            includeDeclaration: true
+        });
+    });
+
+    test('Must find references to parent interface in another interface declaration. Excluding parent interface declaration', async () => {
+        const grammar = `grammar test hidden (WS)
```

Comment:
Minor: The grammars in this file use outdated syntax

```suggestion
        const grammar = `grammar test
        hidden terminal WS: /\\s+/;
```